Update configure-your-profile.md (#990)
fishtown-analytics/docs.getdbt.com: website/docs/dbt-cli/configure-your-profile.md

A typical profile for an analyst using dbt locally will have a target named `dev`, and have this set as the default.

> You may also have a `prod` target within your profile, which creates the objects in your production schema. However, since it's often desirable to perform production runs on a schedule, we recommend deploying your dbt project on a separate machine from your local machine. As such, most dbt users will only have a dev target in their profile on their local machine.

Reviewer: ```suggestion
You may also have a `prod` target within your profile, which creates the objects in your production schema. However, since it's often desirable to perform production runs on a schedule, we recommend deploying your dbt project to a separate machine other than your local machine. Most dbt users only have a `dev` target in their profile on their local machine.
```
